packages/orchestration/src/typeGuards.js

```diff
@@ -76,6 +76,11 @@ export const IBCConnectionInfoShape = M.splitRecord({
   transferChannel: IBCChannelInfoShape,
 });
 
+/**
+ * @type {import('@agoric/internal').TypedPattern<
+ *   import('./cosmos-api.js').CosmosChainInfo
+ * >}
```

Comment:
Do you like this inline import style better? Myself, I always factor them out into separate `@import` declarations.

Reply:
not intentional, just what my IDE did with autocomplete.

Reply:
Then please revise.